src/components/form/form-question.tsx

```diff
@@ -104,3 +119,29 @@ export function FormQuestion({ column }: { column: FormColumn }) {
     </div>
   )
 }
+
+const handleOnInvalid = (
+  e:
+    | FormEvent<HTMLInputElement>
+    | FormEvent<HTMLSelectElement>
+    | FormEvent<HTMLTextAreaElement>
+) => {
+  e.preventDefault()
+  const selectElement = e.target as HTMLSelectElement
+  if (selectElement.value === '') {
+    selectElement.nextElementSibling?.classList.remove('hidden')
+    toast.error('กรุณากรอกคำตอบให้ครบ', { id: 'invalid' })
+  }
+  ;(e.target as HTMLInputElement).nextElementSibling?.classList.remove('hidden')
+  toast.error('กรุณากรอกคำตอบให้ครบ', { id: 'invalid' })
+}
+
+const handleOnInput = (
+  e:
+    | FormEvent<HTMLInputElement>
+    | FormEvent<HTMLSelectElement>
+    | FormEvent<HTMLTextAreaElement>
+) => {
+  e.preventDefault()
+  ;(e.target as HTMLInputElement).nextElementSibling?.classList.add('hidden')
+}
```

Comment:
Defining functions outside of the component is making the code complexity higher.

IMO, we can let this go for now, but we definitely should come back to clean this.
We will move the "form rendering" code to somewhere else anyway.